base/inference.jl

```diff
@@ -3276,44 +3276,6 @@ function replace_getfield!(ast, e::ANY, tupname, vals, sv, i0)
     end
 end
 
-function code_typed(f, types::ANY; optimize=true)
-    types = to_tuple_type(types)
-    code_typed(call, Tuple{isa(f,Type)?Type{f}:typeof(f), types.parameters...}, optimize=optimize)
```

Comment:
@vtjnash Ahh. Maybe that's why it was broken before then. This used to be only defined in `Core.Inference` with a wrapper in `Base`. After moving it out of `Inference`, the `call` is pointing to the correct function now. (Note that `reflection.jl` is also included in `sysimg.jl`)